Utils/Dataflow/pyDKB/dataflow/stage/AbstractStage.py

```diff
@@ -53,17 +70,53 @@ def add_argument(self, *args, **kwargs):
     def parse_args(self, args):
         """ Parse arguments and set dependant arguments if needed.
 
-        Exits with code 2 in case of error (just as ArgumentParser does).
+        Exits in case of error with code:
+            2 -- failed to parse arguments
+            3 -- failed to read config file
         """
         self.ARGS = self.__parser.parse_args(args)
         if not self.ARGS.mode:
             self.args_error("Parameter -m|--mode must be used with value:"
                             " -m MODE.")
 
+        if not self.read_config():
+            self.config_error()
+
     def args_error(self, message):
-        """ Output USAGE, error message and exit. """
+        """ Output USAGE, error message and exit with code 2. """
         self.__parser.error(message)
 
+    def config_error(self, message="Failed to read config file:"):
+        """ Output error message and exit with code 3. """
+        if self._error:
+            message = message + "\n" + str(self._error['exception'])
+        self.output_error(message)
+        sys.exit(3)
+
+    def read_config(self):
+        """ Reads stage custom config file.
```

Comment:
I suggest changing "Reads" to "Read" to be in line with other functions and methods in this file and pyDKB, which are described as "do X", and not "(this function/method) does X".

Reply:
Done: #259
Not in this PR, as it is supposed to be "merge" PR, not "merge-and-improve" one: if some improvement appears here, it won\`t get to `pyDKB` branch before `master` is merged there (again).
Yet after #259 is merged, I can (re-)merge `pyDKB` here, to make sure this change goes to `master` as well.

Reply:
Haven't found anything strange, so I suggest to approve this PR.